src/BIMDataComponents/BIMDataDropdownMenu/BIMDataDropdownMenu.vue

```diff
@@ -110,10 +127,45 @@ export default {
       return {
         "min-width": `${this.width}`,
         "min-height": `${this.height}`,
+        visibility: this.header ? "visible" : "hidden",
       };
     },
   },
+  mounted() {
+    this.mutationObserver = new MutationObserver(this.isHeaderDisplayed);
+    this.mutationObserver.observe(this.$refs.dropdown, {
+      childList: true,
+      subtree: true,
+    });
+  },
   methods: {
+    isHeaderDisplayed() {
```

Comment:
We usually expect a method named `isSomething` to return a boolean and not to set a value somewhere else.